protos/vehicle_data.proto

```diff
@@ -199,14 +202,24 @@ enum ChargingState {
     ChargeStateStopped = 6;
 }
 
+enum DetailedChargeStateValue {
+    DetailedChargeStateUnknown = 0;
+    DetailedChargeStateDisconnected = 1;
+    DetailedChargeStateNoPower = 2;
+    DetailedChargeStateStarting = 3;
+    DetailedChargeStateCharging = 4;
+    DetailedChargeStateComplete = 5;
+    DetailedChargeStateStopped = 6;
+}
+
 // ShiftState are the gear options
 enum ShiftState {
     ShiftStateUnknown = 0;
     ShiftStateInvalid = 1;
     ShiftStateP = 2;
     ShiftStateR = 3;
-    ShiftStateD = 4;
-    ShiftStateN = 5;
+    ShiftStateN = 4;
+    ShiftStateD = 5;
```

Comment:
is this backwards compatible change? I guess yes, since we never used these enums until now?

Reply:
Yes, backwards compatible since not yet used.